packages/dx-react-scheduler/src/plugins/drag-drop-provider.tsx

```diff
@@ -138,9 +138,12 @@ class DragDropProviderBase extends React.PureComponent<
     }
 
     const tableCellElementsMeta = timeTableElementsMeta;
-    const allDayCellsElementsMeta = allDayElementsMeta
-      || { getCellRects: [] }; // not always AllDayPanel exists
 
+    // AllDayPanel doesn't always exist
+    let allDayCellsElementsMeta = allDayElementsMeta;
+    if (!allDayElementsMeta || !allDayElementsMeta.getCellRects) {
+      allDayCellsElementsMeta = { getCellRects: [] };
```

Comment:
I think we should use a ternary operator here, to avoid to use `let` and simplify code.